### ISSUE TITLE
[Bug]: Old copies of privacy.sexy installers are being stored in private var folders

### ISSUE THREAD

**Issue by jvk2** [undergroundwires/privacy.sexy#417]
### Description

I found twelve files such as /private/var/folders/yd/p9cfbbhj68d_92kqm47nt2qr0000gp/privacy.sexy/0.12.5-installer.dmg
Together they occupy 1.61 GB.

### Reproduction steps

These are probably downloaded by the program.

### Expected behavior

I would expect these dmg files to be cleared after installation or a version check.

**Comment by undergroundwires** [comment]
I can reproduce this.

Running `ls /var/folders/*/*/privacy.sexy/*-installer.dmg` gives me:

```
/var/folders/d0/fs0nfsms75j4qqnjp7tf28gw0000gn/privacy.sexy/0.12.8-installer.dmg
/var/folders/d0/fs0nfsms75j4qqnjp7tf28gw0000gn/privacy.sexy/0.12.9-installer.dmg
/var/folders/d0/fs0nfsms75j4qqnjp7tf28gw0000gn/privacy.sexy/0.13.0-installer.dmg
/var/folders/d0/fs0nfsms75j4qqnjp7tf28gw0000gn/privacy.sexy/0.13.1-installer.dmg
/var/folders/d0/fs0nfsms75j4qqnjp7tf28gw0000gn/privacy.sexy/0.13.3-installer.dmg
/var/folders/d0/fs0nfsms75j4qqnjp7tf28gw0000gn/privacy.sexy/0.13.6-installer.dmg
```

Thank you for detecting this.

The path is [received from](https://github.com/undergroundwires/privacy.sexy/blob/db090f369632087285f53c253da24a9a235aa4c8/src/presentation/electron/main/Update/ManualUpdater/Downloader.ts#L51-L60) operating system as **temporary** folder, which is by standard should be cleaned up regularly by the operating system. I don't understand why it does not behave this way, but I will fix it.

I think it would be better to have a more predictable path for privacy.sexy, using application data or something and clear this directory on every launch if any files exist. I will think more about this an fix in a patch.

As temporary solution, you can clean these up by running this command in a terminal window:

```bash
rm -v /var/folders/*/*/privacy.sexy/*-installer.dmg
```

**Comment by jvk2** [comment]
You're welcome. By the way, the only other program I've used recently that stores files here is Graphic Converter. It works for them: I've used it for years and there are only files from the last two days.

**Comment by undergroundwires** [comment]
I did the following changes: privacy.sexy does no longer trust OS to cleanup its temporary directories, stores update installation files at `$HOME/Library/Application Support/privacy.sexy/updates`. It also cleans up contents of this directory on every application launch. It will be released in next patch release.

**Comment by undergroundwires** [comment]
Fix released as part of [0.13.7](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.13.7) 🚀. Feel free to test it and open new issue if there are any other issues. Thank you again.
